commcare_connect/templates/tailwind/layouts/header.html

```diff
@@ -0,0 +1,87 @@
+<header class="fixed top-0 left-16 right-0 bg-gray-100 z-30">
+    <div class="flex items-center justify-between w-full px-4 py-2">
+        <!-- Left Section: Module Name -->
+        <h1 class="text-lg text-brand-deep-purple font-medium">{{ header_title|default:"CommCareConnect" }}</h1>
+
+        <!-- Right Section: Search, Notification, Profile Dropdown -->
+        <div class="flex items-center gap-x-2">
+
+            <!-- Search -->
+            <button class="text-brand-deep-purple focus:outline-none rounded-full hover:bg-white aspect-square w-10">
+                <i class="fa-light fa-magnifying-glass"></i>
+            </button>
+
+            <!-- Notification -->
+            <button class="text-brand-deep-purple focus:outline-none rounded-full hover:bg-white aspect-square w-10 relative">
+                <div class="h-2 w-2 bg-red-500 rounded-full absolute top-2 right-2"></div>
+                <i class="fa-light fa-bell"></i>
+            </button>
+
+            <!-- Organization selector -->
+            <div x-data="{ open: false }" class="relative mx-8">
+                <button @click="open = !open" class="flex items-center gap-2 focus:outline-none">
+                    <div class="text-left">
+                        <p class="text-sm font-medium text-brand-deep-purple cursor-pointer">Organization Names</p>
+                        <p class="text-xs text-gray-400">Network Manager</p>
+                    </div>
+
+                    <!-- Dropdown Toggle Icon -->
+                    <i :class="open ? 'rotate-180' : ''" class="fa-light fa-chevron-down transition-all text-xs"></i>
+                </button>
+
+                <!-- Organization Selector -->
+                <div x-show="open" x-transition.opacity x-cloak class="absolute top-12 right-0 min-w-56 bg-white shadow rounded-lg z-50">
+                    <ul class="p-2 w-full">
+                        <li class="p-2 hover:bg-gray-100 rounded cursor-pointer text-left">
+                            <p class="text-brand-deep-purple text-sm font-medium">Meca Healthcare</p>
```

Comment:
Looks like there is hardcoded data here

Reply:
These are added as placeholder here, they are getting overridden in https://github.com/dimagi/commcare-connect/pull/555